addons/mrp/migrations/15.0.2.1/pre-migration.py

```diff
@@ -0,0 +1,8 @@
+def migrate(cr, version):
+    cr.execute(
+        """
+        UPDATE mrp_production as mp
+           SET date_start = (SELECT MIN(date_start) FROM mrp_workorder WHERE production_id = mp.id)
+         WHERE mp.state != 'draft' AND mp.date_start IS NULL
+        """
+    )
```

Comment:
I don't think we need this migration script, as it won't update existing data anyway.

Reply:
@clesgow Why don't we need a migration script? What about the data from the previous MO? Please let me know?

Reply:
Migration scripts are only triggered when upgrading to a specific version.
In this case, this script will be applied when upgrading to version `15.0`. It won't update data made on this version, nor for later versions.

Reply:
I am not sure if Odoo Online run routine upgrade but this will be good for someone on-premise or odoo.sh who want to run upgrade the module `mrp` to get existing data fixed

Reply:
@clesgow Migration scripts are also triggered when upgrading a single module. _If_ we want this script to take effect we _need_ to bump the module version. The script will be correctly applied.

Reply:
@aj-fuentes Thanks for the clarification, I stand corrected. :slightly_smiling_face: